modules/installation-network-bare-metal.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-network-bare-metal-{context}"]
+= Bare metal network topology requirements
+
+{product-title} requires all nodes to have internet access to pull images
+for platform containers and provide telemetry data to Red Hat.
+
+.Load balancers
+
+Before you install {product-title}, you must provision two load balancers.
+
+[cols="2a,2a,2a,2a,5a",options="header"]
+|===
+
+|Port
+|Machines
+|Internal
+|External
+|Description
+
+|`6443`
+|Bootstrap and control plane
+|x
+|x
+|Kubernetes APIServer
+
+|`22623`
+|Bootstrap and control plane
+|
+|x
+|Machine Config server
+
+|`443`
+|The machines that run the ingress router pods, worker by default
+|x
+|x
+|HTTPS traffic
+
+|`80`
+|The machines that run the ingress router pods, worker by default
+|x
+|x
+|HTTP traffic
+
+|===
+
+[NOTE]
+====
+A working configuration for the ingress router is required for an
+{product-title} cluster. You must configure the ingress router after the control
+plane initializes.
+====
+
+.Connectivity between machines
+
+You must configure the network connectivity between machines to allow cluster
+components to communicate.
+
+////
+.Control plane machines to control plane machines
+[cols="2a,5a",options="header"]
+|===
+
+|`2379`-`2380`
+|etcd server, peer, and metrics ports
+
+|`6443`
+|Kubernetes API
+
+|`9099`
+|Cluster Version Operator
+
+
+|===
+
+.Worker machines to control plane machines
+[cols="2a,5a",options="header"]
+|===
+
+|`2379`-`2380`
+|etcd server and metrics ports
+
+
+|===
+
+
+.All machines to all machines
+[cols="2a,5a",options="header"]
+|===
+
+|`9000`-`9999`
+|Host level services, including the node exporter on ports `9100`-`9101`
+
+|`10249`-`10259`
+|The default ports that Kubernetes reserves
+
+|`10256`
+|openshift-sdn
+
+|`30000`-`32767`
+|Kubernetes NodePort
+
+|===
+////
+
+
+.All machines to all machines
```

Comment:
@abhinavdahiya, @sdodson suggested having all the machines be able to communicate with all other machines. What do you think?

Reply:
I think that should be okay.